### PR TITLE
hub: Build a bottle for Linuxbrew

### PR DESCRIPTION
**Pull request by iMichka**

### CODE REVIEW

**Comment by sjackman** [comment]
```
/bin/sh: 1: col: not found
/bin/sh: 1: groff: not found
Makefile:58: recipe for target 'share/man/man1/hub.1.txt' failed
```
https://circleci.com/gh/Linuxbrew/homebrew-core/17001

`col` is provided by `bsdmainutils`
`groff` is provided by `groff`

Perhaps…
```ruby
if !OS.mac? && build.with?("docs")
  depends_on "bsdmainutils" => :build
  depends_on "groff" => :build
  depends_on "ruby" => :build
end
```

File: Formula/hub.rb [hunk 1]
```diff
@@ -16,11 +16,16 @@ class Hub < Formula
   option "without-docs", "Don't install man pages"
 
   depends_on "go" => :build
-  depends_on "ruby" => :build if build.with?("docs") && !OS.mac?
 
   # The build needs Ruby 1.9 or higher.
   depends_on "ruby" => :build if MacOS.version <= :mountain_lion
```
Comment (sjackman) [inline Formula/hub.rb]: Thanks for this fix, Michka! This line isn't present in Homebrew/core, so I think it can be deleted.

Comment (iMichka) [reply Formula/hub.rb]: It does exist: https://github.com/Homebrew/homebrew-core/blob/master/Formula/hub.rb#L21-L22

Comment (sjackman) [reply Formula/hub.rb]: My mistake. I was looking at an older version. In that case, I'd suggest…
```ruby
  # The build needs Ruby 1.9 or higher.
  depends_on "ruby" => :build if !OS.mac? | MacOS.version <= :mountain_lion
```

Comment (sjackman) [reply Formula/hub.rb]: I wonder whether `ruby` is only needed for building docs, or whether it's always needed.

Comment (sjackman) [reply Formula/hub.rb]: The bottle is relocatable, and `ruby` is a build dep, so it doesn't much matter, since the bottle doesn't depend on `ruby`.

**Pull request closed without merging**